announcement/announcement-api/api/src/java/org/sakaiproject/announcement/api/AnnouncementsUserNotificationHandler.java

```diff
@@ -50,6 +51,9 @@
 @Component
 public class AnnouncementsUserNotificationHandler extends AbstractUserNotificationHandler {
 
+    private static final String ADD_EVENT = AnnouncementService.SECURE_ANNC_ADD;
```

Comment:
why the redeclaration?

Reply:
I thought the code would look clearer and shorter, as those are used multiple times